### ISSUE TITLE
AWC in KDE Plasma 5.12

### ISSUE THREAD

**Issue by t0msk** [kotelnik/plasma-applet-active-window-control#130]
Hello,

I upgraded to KDE Plasma 5.12 (KDE Neon User Edition) and AWC global menu is not working (doesn't show), how is possible to fix it?

Thanks

**Comment by trmdi** [comment]
Use the git version: https://phabricator.kde.org/source/plasma-active-window-control/

**Comment by t0msk** [comment]
What is Phabricator? It is something like git for KDE's apps? And is possible to send a issues on Phabricator?

**Comment by aislanmaia** [comment]
@trmdi why isn't it in the official repo?

**Comment by trmdi** [comment]
@t0msk: Yes. Phabricator is KDE's task management and patch review system. Though, for sending AWC issues, I think this (github) is still the right place.

But it seems that @kotelnik has dropped this project ?

@aislanmaia: It depends on the package maintainer.
